components/layout/fragment.rs

```diff
@@ -3129,6 +3163,32 @@ bitflags! {
         const IS_INLINE_FLEX_ITEM = 0b0000_0001,
         /// Whether this fragment represents a child in a column flex container.
         const IS_BLOCK_FLEX_ITEM = 0b0000_0010,
+        /// Whether this fragment is part of a document in quirks mode.
+        const IS_QUIRKS_MODE = 0b0000_0100,
+        /// Which quirks mode we're in: 0 is quirks, 1 is limited quirks
+        const QUIRKS_MODE = 0b0000_1000,
+    }
```

Comment:
here I decided to integrate it in the bitflags, because I thought fragments are everywhere and I didn't want to use more memory when not needed.